recipes-support/curl/curl/0007-Made-curl_global_init-thread-safe.patch

```diff
@@ -0,0 +1,56 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Ioan-Adrian Ratiu <adrian.ratiu@ni.com>
```

Comment:
Could you update the attribution info, since you modified the .patch contents.

Reply:
Amended.